apps/builder/app/builder/features/style-panel/sections/transitions/transition-utils.ts

```diff
@@ -0,0 +1,17 @@
+import type { TupleValue } from "@webstudio-is/css-engine";
+
+export const convertTupleToStringValue = (layer: TupleValue): string => {
```

Comment:
isn't toValue() converting tuple value to css string? just not sure what's the difference